ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activityTextAnswer.component.ts

```diff
@@ -146,4 +156,18 @@ export class ActivityTextAnswer implements OnChanges, OnInit {
             this.filteredSuggestions$ = of([]);
         }
     }
+
+    ngOnDestroy(): void {
+        this.subscription.unsubscribe();
+    }
+
+    private initTextInputChangesEmitter(): void {
+        this.subscription = this.inputSubject.pipe(
+            debounceTime(300),
```

Comment:
debounce help us here?

Reply:
Yes, it helps to decrease number of patch requests. 

![withDebounce](https://user-images.githubusercontent.com/7396837/113391466-3c40e800-939c-11eb-9629-2ae88c5fae07.png)

![withoutDebounce](https://user-images.githubusercontent.com/7396837/113391489-44992300-939c-11eb-924f-a4e802fe55cd.png)